app/Http/Middleware/SignatureAuth.php

```diff
@@ -22,9 +30,13 @@ public function handle($request, Closure $next)
         $shared_secret = getenv('PLATFORM_SHARED_SECRET');
         $fullURL = $request->fullUrl();
 
-        $verifier = new Verifier($signature, $api_key, $shared_secret, $fullURL, $request->getInputSource()->all());
-
-        if (!$verifier->verified()) {
+        if (!$this->verifier->verified(
+            $signature,
+            $api_key,
+            $shared_secret,
+            $fullURL,
+            $request->getContent()
```

Comment:
This is now the string request contents, not an array decoded from json

Reply:
Oh, Missed that. That's good.